packages/client/components/BillingLeaderActionMenu.tsx

```diff
@@ -45,7 +45,7 @@ const BillingLeaderActionMenu = (props: Props) => {
       if (submitting) return
       submitMutation()
       const variables = {orgId, userId, role}
-      SetOrgUserRoleMutation(atmosphere, variables, {}, onError, onCompleted)
+      SetOrgUserRoleMutation(atmosphere, variables, {onError, onCompleted})
```

Comment:
updated old mutations to use the `StandardMutation<T>` generic so they've all got typesafety now